src/index.html

```diff
@@ -1,12 +1,486 @@
+            href="#"
+            alt="buy-product-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card" data-qa="card">
+        <div class="card__img-container">
+          <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
+
+
+        <div class="card__info-container">
+
+          <div class="card__section card__section--description">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <span class="card__code">
+              Product code: 195434
+            </span>
+          </div>
+
+          <div class="card__section card__section--rating">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star "></div>
+            </div>
+
+            <span class="card__reviews">Reviews: 5</span>
+
+          </div>
+
+          <div class="card__section card__section--pricing">
+            <span class="card__price-label">Price:</span>
+            <span class="card_price-value">$2,199</span>
+          </div>
+
+          <a data-qa="hover"
+
+            class="button card__section button--cta"
+            href="#"
+            alt="buy-product-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card" data-qa="card">
+        <div class="card__img-container">
+          <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
+
+
+        <div class="card__info-container">
+
+          <div class="card__section card__section--description">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <span class="card__code">
+              Product code: 195434
+            </span>
+          </div>
+
+          <div class="card__section card__section--rating">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star "></div>
+            </div>
+
+            <span class="card__reviews">Reviews: 5</span>
+
+          </div>
+
+          <div class="card__section card__section--pricing">
+            <span class="card__price-label">Price:</span>
```

Comment:
```suggestion
            <p class="card__price-label">Price:</p>
```
use semantic `p` tag for text